api/v1alpha1/servicedefaults_types.go

```diff
@@ -144,3 +174,31 @@ func (e ExposeConfig) toConsul() capi.ExposeConfig {
 		Paths:  paths,
 	}
 }
+
+func (e *ExposeConfig) defaultConfig() {
+	for i, path := range e.Paths {
+		if path.Protocol == "" {
+			e.Paths[i].Protocol = "http"
+		}
+	}
+}
+
+func (e ExposeConfig) validate() []*field.Error {
+	var errs field.ErrorList
+	for i, path := range e.Paths {
+		if !strings.HasPrefix(path.Path, "/") {
```

Comment:
If `path.Path` is empty, this will error, but it probably shouldn't since `path` is not required.

Reply:
Is it valid for a ServiceDefault to have ExposePaths with the path being `""`?

Reply:
Yeah it's valid, because you could set only ports without path.